tools/scripts/generateSoundMetadata.rb

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env ruby
+#
+# Given a list of mp3 sounds, generates a matching set of json files that
+# consist of sound metadata for use in the Game Lab/App Lab sound library,
+# ready to upload to S3.
+#
+# Script adapted from generateSingleFrameAnimationMetadata.rb
+#
+# For usage, run ./generateSoundMetadata.rb --help
+#
+# See also: The Sound Library Tech Spec (requires login):
+# https://docs.google.com/document/d/1Oj--H-xwrK3u4A0L5ML73n8XDutsH5n2i8caGRc3NL8/edit
+#
+require 'optparse'
+require 'parallel'
+require_relative '../../deployment'
+require_relative '../../lib/cdo/cdo_cli'
+include CdoCli
+
+begin
+  require 'taglib'
+rescue LoadError
+  puts "You need taglib installed locally to run this script. Run 'brew install taglib' then 'gem install taglib-ruby' and then add 'gem 'taglib-ruby'' to Gemfile and run bundle install. Do not check in changes to Gemfile."
```

Comment:
I think all that's necessary is `brew install taglib` and `gem install taglib-ruby`.  You shouldn't actually need to add anything to the gemfile to use it here.